src/Symfony/Component/Messenger/Tests/EventListener/SendFailedMessageForRetryListenerTest.php

```diff
@@ -108,10 +111,67 @@ public function testEnvelopeIsSentToTransportOnRetry()
         $retryStrategyLocator->expects($this->once())->method('has')->willReturn(true);
         $retryStrategyLocator->expects($this->once())->method('get')->willReturn($retryStategy);
 
+        $logger = $this->createMock(LoggerInterface::class);
```

Comment:
This seems to be out of scope of this particular test case.

Reply:
Right, restored this test to original version.